Sources/PathConfigIdentifiable.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  PathConfigIdentifiable.swift
+//  Calendar
+//
+//  Created by Fernando Olivares on 11/09/23.
+//
+
+import UIKit
+
+/// A covenient way to identify view controllers.
+public protocol PathConfigViewControllerIdentifiable : UIViewController {
+    static var viewControllerPathConfigIdentifier: String { get }
```

Comment:
Not sure if related or not, but maybe? #54

Reply:
I think this change is related, but tangential 🤔

The use case for this protocol is making sure that the "view-controller" property in the Path Config matches a native ViewController. So, for example if we have:

```json
{
  "rules": [
      {
        "patterns": [
          "/recipes/*"
        ],
        "properties": {
            "view-controller": "RecipeViewController",
        }
      },
  ]
}
```

```swift
class RecipeViewController: UIViewController,
                                                PathConfigViewControllerIdentifiable {
    
    static var viewControllerPathConfigIdentifier: String { "RecipeViewController" }
}
```

I'm working on this today. Will post an update soon.